src/vessel-charging/messages/ChargingCompleteMessageParams.test.ts

```diff
@@ -0,0 +1,29 @@
+import MessageParams from './ChargingCompleteMessageParams';
+import { ID, BigInteger } from '../../common-types';
```

Comment:
It does not seem like `ID` and `BigInteger` is being used in these tests. If that is the case, may I suggest removing the import to keep the file lean?

Reply:
Yes! Fixing now and creating new PR.